blockchain/types/account/account_test.go

```diff
@@ -149,7 +149,7 @@ func genSCA() *SmartContractAccount {
 		AccountValueKeyAccountKey:    accountkey.NewAccountKeyLegacy(),
 		AccountValueKeyStorageRoot:   genRandomHash(),
 		AccountValueKeyCodeHash:      genRandomHash().Bytes(),
-		AccountValueKeyCodeInfo:      params.CodeFormatEVM.CodeInfoFromCodeFormat(),
+		AccountValueKeyCodeInfo:      params.CodeInfo(0),
```

Comment:
`params.CodeInfo(0)` is same with `NewCodeInfo(VmVersion0,CodeFormatEVM)`?

Reply:
@KimKyungup yes, it's same.